app/lib/rocketchat/services/getUserInfo.ts

```diff
@@ -0,0 +1,6 @@
+import sdk from './sdk';
+
+export default function getUserInfo(userId: string): Promise<unknown> {
+	// RC 0.48.0
+	return sdk.get('users.info', { userId });
```

Comment:
I have a point just about this kind of method.
If we type the sdk return, we will have to retype this and other methods.
I don't know to what extent it is "healthy" to do this, since every time we change in one place we will have to change in another.